build-system/externs/amp.extern.js

```diff
@@ -718,7 +718,7 @@ let BindEvaluateExpressionResultDef;
 
 /**
  * Options for Bind.rescan().
- * @typedef {{update: (boolean|undefined), fast: (boolean|undefined), timeout: (number|undefined)}}
+ * @typedef {{update: (boolean|string|undefined), fast: (boolean|undefined), timeout: (number|undefined)}}
```

Comment:
nit: this type is awkward. it could make more sense to only support `string` now that there are three modes.

Reply:
I see your point but the current union type makes it easier to check/set the falsy state for `options.update`.